src/db/services/storage/Storage.ts

```diff
@@ -0,0 +1,34 @@
+import admin, { ServiceAccount } from "firebase-admin";
+import { Bucket, UploadResponse } from "@google-cloud/storage";
+import { config, bucketURL } from "../../../../storageConfig";
+
+class Storage {
+  private static bucket: Bucket;
+
+  static getInstane() {
+    if (!this.bucket) {
+      try {
+        admin.initializeApp({
+          credential: admin.credential.cert(config as ServiceAccount),
+          storageBucket: bucketURL,
+        });
+        this.bucket = admin.storage().bucket();
+      } catch (error) {
+        console.log(error);
+      }
+      return this.bucket;
```

Comment:
I think this **return** is redundant, because you already has a return at the end of the method